spec/loader_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'deliver/loader'
+require 'fakefs/spec_helpers'
+
+describe Deliver::Loader do
```

Comment:
Oh, I feel so bad that I didn't write any tests for `deliver` yet, thanks for adding this here :+1: